src/game_renderer/GameRenderer.cpp

```diff
@@ -186,9 +187,33 @@ void GameRenderer::RenderIt(GLFWwindow* window,
   game_state->SetItemsInView(
       GameRenderer::GetObjectsInView(vfplane, level->getTree()));
 
+  // large far for sexy looks
+  P->popMatrix();
+  P->pushMatrix();
+  P->perspective(45.0f, aspect, 0.01f, 1000.0f);
```

Comment:
Oh, so the skybox is part of the spatial data structure now? I thought before we didn't even check it's bounds before drawing, we just assumed it was always drawn.